fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+const url = (path) => `${process.env.REACT_APP_BASE_URL}${path}`;
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+
+
+  useEffect(() => {
+      async function getUser() {
+        try {
+          const response = await fetch(url('/api/todos'));
+          if (!response.ok) {
+            throw new Error(`Network reponse was not OK! Error status:${response.status}`);
+          }
+          const jsonData = await response.json();
+          setTodos(jsonData);
+        } catch (err) {
+          setError(err.message);
+        } finally {
+          setIsLoading(false);
+        }
+      }
+      getUser();
+    /* Use fetch to send a GET request to `/api/todos`.
+     * Once the response JSON is received and parsed,
+     * update state with the received todos.
+     * Handle errors correctly and set `error` to any error or error status code.
+     * Set `isLoading` to false when all network activity is finished.
+     */
+  }, []);
+
+  async function addTodo(newTodo) {
+    try{
+      const response = await fetch(url('/api/todos'), {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify(newTodo)
+      });
+      if (!response.ok) {
+        throw new Error(`Network response was not OK! Error status:${response.status}`);
+      }
+      const jsonData = await response.json();
+      setTodos(todos.concat(jsonData));
+    } catch (err) {
+      setError(err.message);
```

Comment:
If you look at the code that displays `error`, it is expecting the entire error, not just the message.